app/models/search_results.rb

```diff
@@ -1,57 +1,64 @@
 class SearchResults
-  def initialize(files)
-    @items = files.group_by { |item| item.resource_type.first }
+  def initialize(response, files)
+    @response = response
+    @items = files
```

Comment:
We're using the name `items` everywhere else in this class.  Should we rename the temp here and the getter below to match that naming?